airbyte-scheduler/src/main/java/io/airbyte/scheduler/worker_run/TemporalWorkerRunFactory.java

```diff
@@ -52,33 +60,49 @@ public WorkerRun create(Job job) {
     return WorkerRun.create(workspaceRoot, job.getId(), attemptId, createSupplier(job, attemptId));
   }
 
-  @SuppressWarnings("UnnecessaryDefault")
+  @SuppressWarnings({"UnnecessaryDefault", "CodeBlock2Expr"})
```

Comment:
the formatting makes this really hard to read if you don't have the return statements (even though the bracket-less lambda syntax is technically possible in this case).